python_modules/libraries/dagster-dbt/dagster_dbt_tests/cli/test_scaffold.py

```diff
@@ -111,23 +123,26 @@ def test_project_scaffold_command_with_precompiled_manifest(
     _assert_scaffold_defs(project_name=project_name, dagster_project_dir=dagster_project_dir)
 
 
+@pytest.mark.parametrize("use_experimental_dbt_project", [True, False])
 @pytest.mark.parametrize("use_dbt_project_package_data_dir", [True, False])
 def test_project_scaffold_command_with_runtime_manifest(
     monkeypatch: pytest.MonkeyPatch,
     tmp_path: Path,
     dbt_project_dir: Path,
     use_dbt_project_package_data_dir: bool,
+    use_experimental_dbt_project: bool,
 ) -> None:
     monkeypatch.chdir(tmp_path)
 
-    project_name = f"test_dagster_scaffold_runtime_manifest_{use_dbt_project_package_data_dir}"
+    project_name = f"test_dagster_scaffold_runtime_manifest_{use_experimental_dbt_project}"
     dagster_project_dir = tmp_path.joinpath(project_name)
 
     _assert_scaffold_invocation(
         project_name=project_name,
         dbt_project_dir=dbt_project_dir,
         dagster_project_dir=dagster_project_dir,
         use_dbt_project_package_data_dir=use_dbt_project_package_data_dir,
+        use_experimental_dbt_project=use_experimental_dbt_project,
     )
 
     dbt_project_dir = _update_dbt_project_path(
```

Comment:
pretty sure this should get replaced with prepare-for-deployment, check https://github.com/dagster-io/dagster/pull/19925/files for reference

Reply:
I just pushed up [python_modules/libraries/dagster-dbt/dagster_dbt_tests/cli/test_prepare_for_deployment.py](https://github.com/dagster-io/dagster/pull/21089/files#diff-6253b51694e1d8c24fbe934f0b1badc12826b15ddffeef78b68d2bac774ac285), which will test that flow.